.prettierignore

```diff
@@ -1,4 +1,17 @@
 code-examples/protect-page-login/php/composer.lock
 CONTRIBUTING.md
-.github/pull_request_template.md
+.github/*
+tests/*
+src/*
+docs/cli/*
+docs/hydra/cli/*
+docs/keto/cli/*
+docs/kratos/cli/*
+docs/oathkeeper/cli/*
 package-lock.json
+.prettierignore
```

Comment:
Prettier doesn't prettify this file anyways so I think we can leave this entry out.